feature/challenge-reward/data/src/main/java/com/appcoins/wallet/feature/challengereward/data/ChallengeRewardManager.kt

```diff
@@ -0,0 +1,31 @@
+package com.appcoins.wallet.feature.challengereward.data
+
+import android.app.Activity
+import com.appcoins.wallet.feature.challengereward.data.model.ChallengeRewardListener
+import com.fyber.fairbid.ads.OfferWall
+import com.fyber.fairbid.ads.offerwall.ShowOptions
+
+const val APP_ID = "135904"
```

Comment:
whats is that? Shouldn’t it be in the buildConfig?

Reply:
Definitely Should!!! Was one of the first things that i did and totally forgot about it, sorry